### PR TITLE
slack-compose-lintsを導入

### PR DESCRIPTION
**Pull request by Yourein**
## 概要

タイトル通り

### CODE REVIEW

File: phone/features/onboarding/src/main/java/jp/ac/mayoi/onboarding/OnboardingScreen.kt [hunk 1]
```diff
@@ -1,2 +1,41 @@
...
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun kasu_mitaina_composable(
```
Comment (mayoi-design-cicd) [inline phone/features/onboarding/src/main/java/jp/ac/mayoi/onboarding/OnboardingScreen.kt:16]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mayoi-design/maigo_compass_android/blob/2d73b0f6dbe3f135b349348ff0001248f4b9d7a3/phone/features/onboarding/src/main/java/jp/ac/mayoi/onboarding/OnboardingScreen.kt#L16"></span>Composable functions that return Unit should start with an uppercase letter</td>
    </tr>
  </tbody>
</table>

Comment (mayoi-design-cicd) [inline phone/features/onboarding/src/main/java/jp/ac/mayoi/onboarding/OnboardingScreen.kt:16]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mayoi-design/maigo_compass_android/blob/2d73b0f6dbe3f135b349348ff0001248f4b9d7a3/phone/features/onboarding/src/main/java/jp/ac/mayoi/onboarding/OnboardingScreen.kt#L16"></span>Composable functions that return Unit should start with an uppercase letter.They are considered declarative entities that can be either present or absent in a composition and therefore follow the naming rules for classes.See https://slackhq.github.io/compose-lints/rules/#naming-composable-functions-properly for more information.</td>
    </tr>
  </tbody>
</table>

File: phone/features/onboarding/src/main/java/jp/ac/mayoi/onboarding/OnboardingScreen.kt [hunk 2]
```diff
@@ -1,2 +1,41 @@
...
+    modifier: Modifier,
```
Comment (mayoi-design-cicd) [inline phone/features/onboarding/src/main/java/jp/ac/mayoi/onboarding/OnboardingScreen.kt:17]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mayoi-design/maigo_compass_android/blob/2d73b0f6dbe3f135b349348ff0001248f4b9d7a3/phone/features/onboarding/src/main/java/jp/ac/mayoi/onboarding/OnboardingScreen.kt#L17"></span>This @Composable function has a modifier parameter but it doesn't have a default value.See https://slackhq.github.io/compose-lints/rules/#modifiers-should-have-default-parameters for more information.</td>
    </tr>
  </tbody>
</table>

File: phone/features/onboarding/src/main/java/jp/ac/mayoi/onboarding/OnboardingScreen.kt [hunk 3]
```diff
@@ -1,2 +1,41 @@
...
+            text = "実録！カスみたいなComposableを書くとLintは怒ってくれるのか！",
+        )
+        Spacer(modifier = Modifier.size(8.dp))
+        Text(
+            text = "〜slack-android-lintsを入れてみたぜ〜",
+        )
+    }
+}
+
+@Preview
```
Comment (mayoi-design-cicd) [inline phone/features/onboarding/src/main/java/jp/ac/mayoi/onboarding/OnboardingScreen.kt:33]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mayoi-design/maigo_compass_android/blob/2d73b0f6dbe3f135b349348ff0001248f4b9d7a3/phone/features/onboarding/src/main/java/jp/ac/mayoi/onboarding/OnboardingScreen.kt#L33"></span>Composables annotated with @Preview that are used only for previewing the UI should not be public.See https://slackhq.github.io/compose-lints/rules/#preview-composables-should-not-be-public for more information.</td>
    </tr>
  </tbody>
</table>

**Comment by Yourein** [comment]
あーDangerってlevelがerrorのやつを送るとexit codeが1になるんだ

**Pull request merged**